addons/website/models/website_menu.py

```diff
@@ -52,13 +62,19 @@ def create(self, vals):
         '''
         # Only used when creating website_data.xml default menu
         if vals.get('url') == '/default-main-menu':
-            return super(Menu, self).create(vals)
+            res = super(Menu, self).create(vals)
```

Comment:
should be probably call only once at the beginning of the function

remove_absolute_url(vals)